docs/guide/static-deploy.md

```diff
@@ -118,6 +118,14 @@ Now the `preview` command will launch the server at `http://localhost:8080`.
            uses: actions/deploy-pages@v2
    ```
 
+3. If you're deploying to GitHub Pages with a custom domain (eg. `www.example.com`), the following steps may need to be taken to avoid errors.
+
+   Set the `base` in `vite.config.js` to `'/'`.
```

Comment:
I think we can change this part above instead.

> If you are deploying to `https://<USERNAME>.github.io/`, you can omit `base` as it defaults to `'/'`.

Reply:
Thanks for the suggestion! I agree. The initial statement can be revised to include custom domains with GitHub Pages, and to improve clarity. I'll make the adjustments.